package.json

```diff
@@ -13,6 +14,7 @@
     "phoenix": "^1.4.0",
     "phoenix_html": "^2.12.0",
     "ramda": "^0.26.1",
+    "rxjs": "^6.4.0",
```

Comment:
we don't need all of rxjs, and observables is not something unique to rxjs, we could consider using something a bit simpler like https://github.com/zenparsing/zen-observable
https://github.com/sindresorhus/any-observable

Reply:
Cool, will look into it.